### ISSUE TITLE
Issue with Genome Editing Metadata

### ISSUE THREAD

**Issue by hoodriverheather** [Labcas-NIST/Labcas-ui#151]
@yuliujpl There seems to be an issue with some of the Genome Editing Metadata. I just noticed and haven't checked any other collections or datasets.

When i drilled down to this virtual dataset
    https://labcas.jpl.nasa.gov/nist/cd/index.html
 it is showing the Flow Cytometry old metadata (see screenshot):
 ![Image](https://github.com/user-attachments/assets/26f3c615-2fb1-41fc-aa4f-8352043aeea3)

Other virtual datasets look like they contain the correct collection, but all of the metadata is the older version and needs to be updated to the new Version 1. Do you want me to create a new issue just for that? Any of the collection and file metadata updates should carry throughout the system. I can map that out if it would be helpful :) I completely forgot about this level.
![Image](https://github.com/user-attachments/assets/8530876f-5118-4935-a310-4cb98039a6db)

**Comment by yuliujpl** [comment]
I'm looking into it, I suspect that the virtual hierarchy view needs to be updated!

**Comment by hoodriverheather** [comment]
@yuliujpl let me know if you want me to provide you with screenshots of the metadata updates. :)

**Comment by yuliujpl** [comment]
Changed! Here is the mapping I used! virtual_hierarchy_dataset_displayfields_keys is the actual keys in solr, and virtual_hierarchy_dataset_displayfields_labels is the change in display labels I map to!

"virtual_hierarchy_dataset_displayfields_keys":"DatasetDescription,WorkingGroupDesignation,LeadPoC,LeadPoCEmail,OwnerPrincipal,ProtocolHandle,SourceID,ParentCellLine,MammalianCellCode,InstrumentHandle,ExpansionType,StudyType,DataCustodian,DataCustodianName,DataCustodianEmail,BlindedDataset,AssayType,CoreCapabilities,PrimaryFocusAreas",

  "virtual_hierarchy_dataset_displayfields_labels":"Dataset Description,Working Group Designation,Principal Contact Name,Principal Contact Email,Owner Principal,Protocol Handle,Source ID,Parent Cell Line,Mammalian Cell Code,Instrument Handle,Expansion Type,Data Category,Data Custodian Name,Data Custodian Name,Data Custodian Email,Blinded Dataset,Assay Type,Core Capabilities,Primary Focus Areas",
